Gemfile

```diff
@@ -1,21 +1,22 @@
 source "https://rubygems.org"
 git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 
-ruby "3.2.3"
+ruby "3.2.2"
```

Comment:
Update your ruby version to 3.2.3 so others does not need to change it when they pull the changes. :)